qdrant/v1.0.x/points.md

```diff
@@ -499,7 +499,7 @@ POST /collections/{collection_name}/points/scroll
     },
     "limit": 1,
     "with_payload": true,
-    "with_vector": false
```

Comment:
It is `with_vector` in REST API, we will make alias first

Reply:
implemented in qdrant/qdrant#1697 👍